neard/src/cli.rs

```diff
@@ -300,6 +300,10 @@ pub(super) struct InitCmd {
     /// from genesis configuration will be taken.
     #[clap(long)]
     max_gas_burnt_view: Option<Gas>,
+    /// Dump epoch config from the given protocol version onwards.
+    /// If the argument is provided with no value, the latest protocol version will be used.
+    #[clap(long)]
+    dump_epoch_config: Option<Option<ProtocolVersion>>,
```

Comment:
option<option<..>> is a bit nonintuitive, for example how will it look like in the command line for setting it to None vs. Some(None) vs. some value?
any chance to simplify this to a single option?

Reply:
Yeah, good point. Even without CLI argument, `Option<Option<...>>` is always hard to reason about to me. Can we use enum with three values, like None + Since(ProtocolVersion) + Latest?

Reply:
Good point. I will do it with enum.